hol-light/readme.md

```diff
@@ -0,0 +1,105 @@
+# HOL-Light
+
+HOL-Light is proof assistant by John Harrisson in the tradition of LCF.
+You can find more information at https://www.cl.cam.ac.uk/~jrh13/hol-light/.
+
+HOL-Light's old-school style prompts for a lot of uppercase. That makes
+it sound more authoriative.
+
+Like in the HOL4 proof, we are using functions on lists from the
+standard library to define `LEFTPAD`. There is no nice infix syntax
+for list concatenation by default; I guess because lists are not really
+used a lot since HOL-Light is geared towards maths.
```

Comment:
Link the reference to another proof, for convenience:

```suggestion
Like in the [HOL4 proof](../hol4/),
we are using functions on lists from the standard library to define `LEFTPAD`.
There is no nice infix syntax for list concatenation by default;
I guess because lists are not really used a lot since HOL-Light is geared towards maths.
```